modules/cdp-engine/src/lib.rs

```diff
@@ -1316,6 +1321,12 @@ impl<T: Config> RiskManager<T::AccountId, CurrencyId, Balance, Balance> for Pall
 				debit_value >= T::MinimumDebitValue::get(),
 				Error::<T>::RemainDebitValueTooSmall,
 			);
+
+			// check the minimum_collateral_value
+			ensure!(
```

Comment:
check check should only happen when debit is zero
otherwise we enforce minimal debit which will indirectly enforce minimal collateral

Reply:
Moved the collateral check to the case where debit is 0.
updated unit tests accordingly